### DISCUSSION TITLE
Is there any reason why the `dataTransfer` object is missing from `state` in all useXXX hooks?

### DISCUSSION THREAD

**Discussion by ghost** [pmndrs/use-gesture#540] [Q&A]
I'm trying to create a drag and drop feature with dynamic lists and came across several sources explaining how to use the `dataTransfer` object with the `onDragXXX` events. I've noticed that the `dataTransfer` object is not present on any of them and I was wondering why and if I'm missing something.

**Comment by dbismut** [comment]
Yes, this is not a drag and drop library, and it's not using `dragstart / dragstop` events. The `useDrag` handler is just to move things around.

**Reply by ghost** [reply]
Oh, that's a bummer. Thanks for the reply!
